lib/feature/detail_card/widget/detail_card_list_comment_widget.dart

```diff
@@ -86,7 +86,8 @@ class DetailCardListCommentWidget extends ConsumerWidget {
                   shrinkWrap: true,
                   itemCount: bloc.listCommentFragmentsSubject.value.length,
                   itemBuilder: (context, i) {
-                    final comment = bloc.listCommentFragmentsSubject.value[i];
+                    final comment = bloc.listCommentFragmentsSubject.value[
+                        bloc.listCommentFragmentsSubject.value.length - 1 - i];
```

Comment:
```suggestion
                    final comment = bloc.listCommentFragmentsSubject.value[
                        bloc.listCommentFragmentsSubject.value.length - 1 - i];
```
đoạn này dùng reversed được mà

Reply:
tôi thử .reversed rồi mà không được thế là dùng cách này luôn cho nhanh

Reply:
![image](https://github.com/Capstone-Project-Fpoly/task-manager-app/assets/102580970/880801cc-6cc1-426a-9f71-28bdd458c62e)
dúng chỗ init này này

Reply:
thử luôn rồi ô
-để tối tôi thử lại xem, tại lúc tối qua code hơi vội